TypeCobol.Test/Parser/Programs/Cobol85/ExecSql/ExecSqlWithWhenEverStatement.rdzSQL.txt

```diff
@@ -0,0 +1,20 @@
+﻿--- Diagnostics ---
+Line 10[14,21] <27, Error, Syntax> - Syntax error : mismatched input 'END-EXEC' expecting {':', user defined word} RuleStack=codeElement>whenEverStatement>sqlGotoHostLabel,  OffendingSymbol=[14,21:END-EXEC]<END_EXEC>
+
+--- Sql Statements ---
+line 6: WhenEverStatement
+- ExceptionCondition = Sqlerror
+- NextStatementType = Goto
+- HostLabel = ENDDATA
```

Comment:
I wanted name changes, see [my remark](https://github.com/TypeCobolTeam/TypeCobol/issues/2246#issuecomment-1148476563) in issue.